helm/istio_cni/templates/daemonset.yaml

```diff
@@ -45,25 +45,25 @@ spec:
           operator: Exists
         - effect: NoExecute
           operator: Exists
-      serviceAccountName: istio-cni-{{ .Release.Namespace }}
+      serviceAccountName: istio-cni
       # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
       # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
       terminationGracePeriodSeconds: 0
-{{- if .Values.global.priorityClassName }}
-      priorityClassName: "{{ .Values.global.priorityClassName }}"
+{{- if .Values.priorityClassName }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
 {{- end }}
-{{- if .Values.global.imagePullSecrets }}
+{{- if .Values.imagePullSecrets }}
       imagePullSecrets:
```

Comment:
Can't we fallback to global?

Reply:
There is no spoon. :)

There is no global, actually. This is not a sub-chart anymore and is deployed separately from everything else.